src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,7 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
```

Comment:
does this method work correctly?
Which result should we get with this parameters: 1, 1, 1, 2?